nion/swift/ExportDialog.py

```diff
@@ -261,15 +261,29 @@ class UnitType(enum.Enum):
 
 
 class ExportSizeModel(Observable.Observable):
-    def __init__(self, display_item: DisplayItem.DisplayItem) -> None:
+    unit_to_str_map = {
+        UnitType.PIXELS: "pixels",
+        UnitType.INCHES: "inches",
+        UnitType.CENTIMETERS: "centimeters"
+    }
+
+    str_to_unit_map = {
+        "pixels": UnitType.PIXELS,
+        "inches": UnitType.INCHES,
+        "centimeters": UnitType.CENTIMETERS
+    }
+
+    def __init__(self, display_item: DisplayItem.DisplayItem, units_model: UserInterface.StringPersistentModel) -> None:
```

Comment:
unit_model should be a more general type here, e.g. Model.PropertyModel[str] - that would allow us to change the model used in the future without having to alter this function